tests/pytests/test_expire.py

```diff
@@ -107,14 +107,19 @@ def expireDocs(env, isSortable, expected_results):
 
         conn.execute_command('PEXPIRE', 'doc1', 1)
         # ensure expiration before search
-        time.sleep(0.01)
+        time.sleep(0.5)
```

Comment:
with 0.01 and SET-ACTIVE-EXPIRE=**1** the keys are not expiring. My guess is that the time interval in redis to check eagerly for expired keys is larger.
For the parts where SET-ACTIVE-EXPIRE=**0** 0.01 should be enough.
 I just wanted to make sure that we pass the time redis would have clean the key (if this flag wasn't set), so in case it is discarded from the results it is discarded because it was found to be expired during the query. But I guess it is on redis to check that their DEBUG commands actually work :)

Reply:
OK, not related to this PR but basically [expire accuracy](https://redis.io/commands/expire/#expire-accuracy) is supposed to be from 0 to 1 milliseconds (`PEXPIRE` is the same as `EXPIRE` in this regard), and I would expect it to behave this way at least when **SET-ACTIVE-EXPIRE=1** (the default mode)